### PR TITLE
Adding landr components

### PR DESCRIPTION
**Pull request by andrewnhem**
This feature adds the components to create a Landr site for balenaSense.
No breaking changes were made, but readme on project repo might look a bit different.

New services added:
- adding and refactoring components to build page with Landr

Change-type: minor
Signed-off-by: Andrew Nhem <andrewn@balena.io>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,116 @@
...
-Contributions are welcome!
+balenaSense is one of the flagship projects created and maintained by balenaLabs. Community contribution is greatly encouraged and appreciated. Use the following guidelines to successfully and sustainably contribute your hard work to take balenaSense to the next level.
 
-## Semantic versioning and commit messages
 
-The balenaSense version numbering adheres to [Semantic Versioning](http://semver.org/). The following
-header/row is required in the body of a commit message, and will cause the CI build to fail if absent:
+## Overview
+
+Make sure you checkout our [architecture][ARCHITECTURE] guide, which aims to explain how all pieces fit together. It's a good starting point for understanding how we designed and built balenaSense.
```
Comment (tmigone) [inline CONTRIBUTING.md:8]: I don't think we have a public architecture guide for sense, right? Perhaps we can leave this but add an issue to have it done later.

File: docs/04-device-support.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+# Supported devices
```
Comment (tmigone) [inline docs/04-device-support.md:2]: This section should be updated for sense

File: CONTRIBUTING.md (hunk 1 above)
Comment (andrewnhem) [reply CONTRIBUTING.md:8]: I agree. I meant to just have this as placeholder. If it's misleading/not helpful, I can issue it and remove it. Let me bring up the architecture guide in this week's Labs call to see how soon we can populate this. Thanks @tmigone !

File: docs/04-device-support.md (hunk 2 above)
Comment (andrewnhem) [reply docs/04-device-support.md:2]: See comment above. Maybe it's ok for MVP? Ideally, I'd like someone with a bit more hardware/testing exp to help inform this. I've only ever used Sense with a pi3 and 4 :p

Comment (tmigone) [reply docs/04-device-support.md:2]: @chrisys most likely will have an idea

File: CONTRIBUTING.md (hunk 1 above)
Comment (tmigone) [reply CONTRIBUTING.md:8]: yeah true. for sound we don't have a public one either lol.

**Pull request closed without merging**